src/file_location.py

```diff
@@ -63,7 +61,8 @@ def resolve(self) -> list[Path]:
         p = Path(ROOT_PATH)
         return list(p.glob(self.substituted_path))
 
-    def substitute(self) -> str:
```

Comment:
Same question as above for substitutes.

e.g `dfm substitute <some string> .... <arguments to pass subs>`

Reply:
Same as above